### PR TITLE
INTLY-2955 - enable extra params to PoC install pipeline

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

File: jobs/openshift/cluster/integreatly/install/Jenkinsfile [hunk 1]
```diff
@@ -77,6 +77,29 @@ node('cirhos_rhel7') {
...
+            openshift_password: ${installerOptions.clusterAdminPassword}
+            integreatly_project_install_scm_url: ${installerOptions.installationGitUrl}
+            integreatly_project_install_scm_branch: ${installerOptions.installationGitBranch}
+            eval_seed_users_count: ${installerOptions.userCount}
+            eval_self_signed_certs: '${installerOptions.selfSignedCerts}'
+            integreatly_inventory_source_aws_credentials: ${installerOptions.awsCredentials}"""
+
+            // append additional parameters    
+            for(String param : additionalParams) {
+                extraVars = extraVars + "\n            " + param.trim()
```
Comment (tremes) [inline jobs/openshift/cluster/integreatly/install/Jenkinsfile]: This is extra ugly, but I don't know how to do that better. I tried to use map but I had some issues with serialization (not sure why)

**Comment by tremes** [comment]
Example run is https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/OpenShift/job/openshift-cluster-integreatly-install-tremes/60/console

File: jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml [hunk 2]
```diff
@@ -41,6 +41,9 @@
           name: 'dryRun'
           default: false
           description: '[OPTIONAL][Test] Dry run only, only prints what it would do!'
+      - string:
+          name: 'additionalParameters'
+          description: '[OPTIONAL] Comma separated list of additional parameters e.g amq_streams: true.'
```
Comment (mikenairn) [inline jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml]: Would a text type parameter work better here? Could split on new line for each param.

Comment (tremes) [reply jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml]: Thanks. It sounds good. I'll check

Comment (tremes) [reply jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml]: Updated. It looks better and the params are separated by a new line. Thanks

**Pull request merged**